.github/workflows/maven.yml

```
@@ -31,14 +31,33 @@
         java-version: '23'
         distribution: 'temurin'
         cache: maven
+    - name: Get External jars
+      working-directory: ${{ runner.temp }}
+      env:
+        SQUID_VERSION: 0.10.1
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:101: Double quote to prevent globbing and word splitting

(shellcheck)